aries-site/src/pages/components/search.js

```diff
@@ -0,0 +1,193 @@
+            docs="https://v2.grommet.io/textinput?theme=hpe#props"
+            figma="https://www.figma.com/file/KKHWJN4GAI0Mq5yh0snDT6/HPE-Search-Component?node-id=265%3A112"
+          >
+            <SearchExample />
+          </Example>
+        </Subsection>
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Guidance">
+          <SubsectionText>
+            Search inputs provide a powerful means for users to access content
+            quickly and efficiently.
+          </SubsectionText>
+          <SubsectionText>
+            Many search best practices extend "behind the scenes," such as how
+            query strings are matched to relevant content and to the efficiency
+            in which that information is retrieved.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Auto-suggestions" level={3} gap="small">
+          <Box width={{ max: 'large' }}>
+            <BulletedList
+              level={3}
+              items={[
+                <Text>
+                  Provide effective{' '}
+                  <Anchor href="#search-with-auto-suggestions">
+                    auto-suggestions
+                  </Anchor>{' '}
+                  whenever possible.
+                </Text>,
+                `Auto suggestions accelerate search and allow users to focus
+                on task completion.`,
+              ]}
+            />
+          </Box>
+        </Subsection>
+        <Subsection name="Returning search results" level={3} gap="small">
+          <Box width={{ max: 'large' }}>
+            <BulletedList
+              level={3}
+              items={[
+                `Use "live filtering." When results are visibile and quick to 
+                filter, results should be filtered as the user types.`,
+                `Sort results by highest relevance. Relevance can be 
+                determined by keyword matching, however, to really elevate user 
+                experience, consider implementing probability based 
+                approaches.`,
+                `Persist users' original query. This eliminates the need to 
+                re-type and assists query reformation if needed.`,
+                `Persist users' recently used search queries. Remembering the 
+                user's previous efforts respects their time and serves as 
+                "short cuts" for returning to previously visited content.`,
+              ]}
+            />
+          </Box>
+        </Subsection>
+        <Subsection name="Accessibility" level={3} gap="small">
+          <SubsectionText>
+            Search inputs should be accessible. Ensure users have the ability to
+            return results using their keyboard as well as clicking a button.
+          </SubsectionText>
+          <SubsectionText>
+            Inputs should always be accompanied by labels for accessibility. An
+            icon may serve as a label if: 1) the icon meaning is well
```

Comment:
can we use a bulleted list maybe to have the 1) and 2) stick out instead of inline with sentence ?

Reply:
Implemented.